### ISSUE TITLE
Still not localized 1.3.5beta build

### ISSUE THREAD

**Issue by eithallkats** [DukeDukeEm/Translation_ShortestTripToEarth_German#97]
remained issue at 1.3.5prebuild (all of them are reported on past posts)
star rank, search point Name2, event log
![x20210526181701_1](https://user-images.githubusercontent.com/84577240/119648870-eb5de500-be5c-11eb-8aec-9a434084636e.jpg)
item names when try to craft
![x20210526180210_1](https://user-images.githubusercontent.com/84577240/119649170-4a235e80-be5d-11eb-9e3b-d4638a11cc09.jpg)
continued development
![x20210526173417_1](https://user-images.githubusercontent.com/84577240/119649514-ae462280-be5d-11eb-93fb-75732ef9be99.jpg)
And, if possible, the popup window of Disclaimers should have darker background (or thick letter)
it's hard to read.

**Comment by DukeDukeEm** [comment]
![image](https://user-images.githubusercontent.com/69652463/119845602-b491d880-bf09-11eb-9e05-82e81cfba6fe.png)
I'm able to translate that EXIT button, It's in the last line of 
Shop panel tabbed.prefab

**Comment by eithallkats** [comment]
Yes, It was a misleading report. sorry. I just tell about we set remained "EXIT" as "EXIT" in our tsv file, so it's not problem it looks unchanged in the photo.

**Comment by eithallkats** [comment]
we set google translate before human work at the test file. and all those texts in the tsv file has (仮)(temporally)Japanese letters.
So, the English words in game are missing localization except the ones we set the words back as English.
This "EXIT" was what we set back.

**Comment by Duke-M-commits** [comment]
all of these have been fixed by the devs in the meantime, except the points of interest on the starmap, that issue is still there and has its own issue here: #26
